ai/smart-wallet-example/smart-wallet-core-solidty/.env.example

```diff
@@ -0,0 +1,6 @@
+DEPLOYER_WALLET_PRIVATE_KEY=
+WALLET_PRIVATE_KEY=0xe50b01b94e4c585be608a22253689009ae0705f33e794a16bf40bc4c958ffe7c
+#WALLET_PRIVATE_KEY=0x7c13413162408f08e751d89e27c59f11b81d5dc4a2fb39a3f611ebf9f5b598ee
```

Comment:
can be removed ?

Reply:
this is critical

Reply:
remove , and use another way ? (env?)

Reply:
I copied the repository from here https://github.com/thomas-nguy/hackaton-otk/blob/master/.env.example

@thomas-nguy what keys are these? Just testing keys? I'll remove them from here. If these keys are just testing keys from random wallets we can just remove and update them. If not, We'll want to rebase to remove the PR and commit again

Reply:
Yes these are testing key from the auto generated file
https://github.com/thomas-nguy/hackaton-otk/blob/master/generated-wallet.txt

Safe to remove

Reply:
Ok, I dropped the commits just in case.